packages/graph/hash_graph/lib/graph/src/lib.rs

```diff
@@ -38,6 +38,9 @@
     clippy::module_name_repetitions,
     reason = "This encourages importing `as` which breaks IDEs"
 )]
-
+#[allow(
+    clippy::must_use_candidate,
+    reason = "#[must_use] adds boilerplate code, which isn't worth it"
+)]
```

Comment:
Thoughts? @Alfred-Mountfield @thehabbos007

Reply:
Have you found false positives so far? It sounds like they did a lot of work on reducing them, checking for interior mutability and things

Reply:
And does it make sense for those methods to have `#[must_use]`?

Reply:
They are getter functions. I'm unsure if we want that. We can still disable the lint later.

Reply:
Why would we want a `get` method without the return value being used? Perhaps it is a good warning

Reply:
I added the attributes in 4b966a1baff879fa985e865e597cff2abc2d8d75